docs/_extra/api-reference/schemas/pagination-meta.yaml

```diff
@@ -3,6 +3,5 @@ PaginationMeta:
   type: object
   properties:
     total:
-      description: "The total number of memberships in the group (across all pages)."
```

Comment:
This is used for both the list-group-memberships and list-group-annotations APIs so this description was not correct. Also I don't think this gets rendered anywhere in the resulting docs anyway.